src/azure-cli-core/setup.py

```diff
@@ -39,6 +39,7 @@
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
     'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
```

Comment:
This should also be added to 

https://github.com/Azure/azure-cli/blob/b616adffc346d49fbf851f87dc8497ff28a5ad55/src/azure-cli/setup.py#L48

https://github.com/Azure/azure-cli/blob/a5198b578b17de934e15b1c92e369e45323e9658/src/azure-cli-telemetry/setup.py#L22

https://github.com/Azure/azure-cli/blob/a5198b578b17de934e15b1c92e369e45323e9658/src/azure-cli-testsdk/setup.py#L21